new_samples/workflows/hello_world_workflow.go

```diff
@@ -0,0 +1,43 @@
+package workflows
+
+import (
+	"context"
+	"fmt"
+	"go.uber.org/cadence/activity"
+	"go.uber.org/cadence/workflow"
+	"go.uber.org/zap"
+	"time"
+)
+
+type sampleInput struct {
+	Message string `json:"message"`
+}
+
+// HelloWorldWorkflow greets the caller.
+func HelloWorldWorkflow(ctx workflow.Context, input sampleInput) error {
+	ao := workflow.ActivityOptions{
+		StartToCloseTimeout:    time.Minute,
+		ScheduleToStartTimeout: time.Minute,
```

Comment:
```suggestion
		ScheduleToStartTimeout: time.Minute,
		StartToCloseTimeout:    time.Minute,
```
matches definition + that's the order they apply in (as the sequence is schedule -> start -> close).

Reply:
auto committed by accepting suggestion